pykechain/client.py

```diff
@@ -1689,7 +1708,8 @@ def clone_scope(
         if asynchronous:
             return None
 
-        cloned_scope = Scope2(response.json()['results'][0], client=source_scope._client)
+        data = response.json()['results'][0]
```

Comment:
free the data!!! refactor it back inside the `Scope2(... )` creation.

Reply:
woops, was trying to resolve the to-do but forgot to roll this back